tidy3d/components/simulation.py

```diff
@@ -1784,33 +1782,32 @@ def subsection(
                     new_bounds[0][dim] = grids_1d[dim][grid_inds[dim][0]]
                     new_bounds[1][dim] = grids_1d[dim][grid_inds[dim][1]]
 
-        # if symmetry is not overriden we inherit it from the original simulation where is needed
-        if symmetry is None:
```

Comment:
Actually this is another case where I slightly prefer the previous handling. Previously if the user specified symmetry, then it wouldn't be inferred or expanded at all. In the new handling, there is no way to prevent automatic symmetry expansion.

Reply:
thanks!